src/worker.rs

```diff
@@ -45,6 +46,40 @@ pub struct Worker {
     logger: Logger,
 }
 
+fn get_git_sha(
+    revision: i32,
+    name: &str,
+    revision_mapping: &mut HashMap<i32, Option<String>>
+) -> Option<String> {
+    match revision_mapping.entry(revision) {
+        Vacant(entry) => {
+            let get_sha = Command::new("svn")
+                .arg("propget")
+                .arg("git-commit")
+                .arg("--revprop")
+                .arg("-r")
+                .arg(revision.to_string())
+                .arg(format!("{}/{}/trunk", ORGANIZATION_ROOT, name))
+                .output();
+            
+            match get_sha {
+                Ok(Output { status, ref stdout, .. }) if status.success() => {
+                    entry.insert(Some(format!("{}", String::from_utf8_lossy(stdout))));
+                },
+                Ok(Output { stderr, .. }) => {
```

Comment:
I still think for both these issues we should be logging and then inserting `None`.